docs/sources/configuration/_index.md

```diff
@@ -2105,6 +2105,14 @@ compacts index shards to more performant forms.
 # CLI flag: -boltdb.shipper.compactor.delete-request-cancel-period
 [delete_request_cancel_period: <duration> | default = 24h]
 
+# Constrain the size of any one delete request. When a delete request > delete_max_query_range
```

Comment:
nit (obs: my english is bad so if that's correct just ignore this :rofl: )
```suggestion
# Constrain the size of a delete request. When a delete request > delete_max_query_range
```

Reply:
Yours is way clearer, thanks!